app/questionnaire/questionnaire_schema.py

```diff
@@ -88,8 +87,11 @@ def get_title_for_section(self, section_id):
     def get_show_on_hub_for_section(self, section_id):
         return self._sections_by_id.get(section_id).get("show_on_hub", True)
 
-    def get_summary_for_section(self, section_id: str):
-        return self._sections_by_id.get(section_id).get("summary")
+    def get_summary_for_section(self, section_id: str) -> dict:
```

Comment:
```suggestion
    def get_summary_for_section(self, section_id: str) -> Mapping:
```

Reply:
This is a literal dict right?

Reply:
Correct, but best practice dictates using Mapping https://docs.python.org/3/library/typing.html#typing.Dict - Mapping is also used elsewhere, so the use of `dict` is inconsistent